W-17747674-http-cookie-kt (#2797)
mulesoft/docs-connectors: http/1.10/modules/ROOT/pages/http-documentation.adoc

** AUTO
** ALWAYS
** NEVER |  Defines if the request should be sent using streaming or not. If this attribute is not present, the behavior will depend on the type of the payload (it will stream only for InputStream). If set to true, it will always stream. If set to false, it will never stream. As streaming is done the request will be sent user Transfer-Encoding: chunked. |  AUTO |

> | Enable Cookies a| Boolean |  If true, cookies received in HTTP responses will be stored, and sent in subsequent HTTP requests. 

Reviewer: ```suggestion
| Enable Cookies a| Boolean |  If true, cookies received in HTTP responses are stored and sent in subsequent HTTP requests. 
```

When an `http:request-config` element has cookies enabled, a single `CookieManager` is created for each instance of the `request-config`. If multiple `http:request` elements in a Mule application reference the same `http:request-config`, they all share the same `CookieManager` instance.

> Note that if you have a dynamic configuration, a new `http:request-config` instance is created for each parameterization, for details refer to the xref:mule-sdk::static-dynamic-configs.adoc#dynamic-configuration-lifecycle[dynamic configurations lifecycle] documentation. Therefore, each parameterization has its own `CookieManager` instance.

Reviewer: ```suggestion
If you have a dynamic configuration, a new `http:request-config` instance is created for each parameterization. Therefore, each parameterization has its own `CookieManager` instance. For details, see xref:mule-sdk::static-dynamic-configs.adoc#dynamic-configuration-lifecycle[]. 
```

Or make it a real note

> This feature can be useful, for example, to have a separate `CookieManager` instance for each event. |  true |

Reviewer: Sentence feels a bit off. Is it useful to have a separate cookie manager instance? Like how does it relate to this feature.
